lib/main.dart

```diff
@@ -36,7 +38,8 @@ class RouteCellWidget extends StatelessWidget {
         title: Wrap(
           spacing: 10,
           children: [
-            Text(place.thumbnail, style: TextStyle(fontWeight: FontWeight.bold)),
+            Text(place.thumbnail,
```

Comment:
Why have you changed that?

Reply:
eto ne ya, its Ctrl+Alt+L(Reformat Code)